server.js

```diff
@@ -6,7 +6,7 @@ let path = require( 'path' ),
     express = require( 'express' ),
     app = express();
 
-app.use( express.static( 'public' ) );
+app.use( express.static( path.join(__dirname, "public") ) );
```

Comment:
What is the benefit of a change like this?

Reply:
> it’s safer to use the absolute path of the directory that you want to serve:

From the expressjs docs.